spec/dummy/db/schema.rb

```diff
@@ -11,72 +11,72 @@
-    t.integer  "active_gallery_id"
-    t.string   "image"
+  create_table "active_gallery_photos", force: :cascade do |t|
+    t.integer  "active_gallery_id", limit: 4
+    t.string   "image",             limit: 255
     t.datetime "created_at"
     t.datetime "updated_at"
   end
 
   add_index "active_gallery_photos", ["active_gallery_id"], name: "index_active_gallery_photos_on_active_gallery_id", using: :btree
 
-  create_table "article_translations", force: true do |t|
-    t.integer  "article_id", null: false
-    t.string   "locale",     null: false
+  create_table "article_translations", force: :cascade do |t|
+    t.integer  "article_id", limit: 4,   null: false
+    t.string   "locale",     limit: 255, null: false
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.string   "name"
+    t.string   "name",       limit: 255
   end
 
   add_index "article_translations", ["article_id"], name: "index_article_translations_on_article_id", using: :btree
   add_index "article_translations", ["locale"], name: "index_article_translations_on_locale", using: :btree
 
-  create_table "articles", force: true do |t|
-    t.integer  "author_id"
+  create_table "articles", force: :cascade do |t|
+    t.integer  "author_id",         limit: 4
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.integer  "author_position"
+    t.integer  "author_position",   limit: 4
     t.datetime "publish_time"
-    t.integer  "active_gallery_id"
-    t.integer  "status"
+    t.integer  "active_gallery_id", limit: 4
+    t.integer  "status",            limit: 4
   end
 
   add_index "articles", ["active_gallery_id"], name: "index_articles_on_active_gallery_id", using: :btree
   add_index "articles", ["author_id"], name: "index_articles_on_author_id", using: :btree
 
-  create_table "author_rubrics", force: true do |t|
-    t.integer "author_id", null: false
-    t.integer "rubric_id", null: false
-    t.integer "position"
+  create_table "author_rubrics", force: :cascade do |t|
+    t.integer "author_id", limit: 4, null: false
+    t.integer "rubric_id", limit: 4, null: false
+    t.integer "position",  limit: 4
   end
 
-  create_table "author_translations", force: true do |t|
-    t.integer  "author_id",  null: false
-    t.string   "locale",     null: false
+  create_table "author_translations", force: :cascade do |t|
+    t.integer  "author_id",  limit: 4,   null: false
+    t.string   "locale",     limit: 255, null: false
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.string   "photo"
+    t.string   "photo",      limit: 255
   end
 
   add_index "author_translations", ["author_id"], name: "index_author_translations_on_author_id", using: :btree
   add_index "author_translations", ["locale"], name: "index_author_translations_on_locale", using: :btree
 
-  create_table "authors", force: true do |t|
-    t.string   "name"
+  create_table "authors", force: :cascade do |t|
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.